_field-types/index.md

```diff
@@ -84,84 +120,29 @@ POST sample-index1/_mapping
   }
 }
 ```
+{% include copy-curl.html %}
 
 You cannot change the mapping of an existing field, you can only modify the field's mapping parameters. 
 {: .note}
 
----
-## Mapping example usage
+## Mapping parameters
 
-The following example shows how to create a mapping to specify that OpenSearch should ignore any documents with malformed IP addresses that do not conform to the [`ip`]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/ip/) data type. You accomplish this by setting the `ignore_malformed` parameter to `true`.
+Mapping parameters are used to configure the behavior of fields in an index. See [Mappings and field types]({{site.url}}{{site.baseurl}}/field-types/) for more information.
 
-### Create an index with an `ip` mapping
+## Mapping limit settings
 
-To create an index, use a PUT request:
+OpenSearch has certain limits or settings related to mappings, such as the settings listed in the following table. Settings can be configured based on your requirements. 
 
-```json
-PUT /test-index 
-{
-  "mappings" : {
-    "properties" :  {
-      "ip_address" : {
-        "type" : "ip",
-        "ignore_malformed": true
-      }
-    }
-  }
-}
-```
-
-You can add a document that has a malformed IP address to your index:
-
-```json
-PUT /test-index/_doc/1 
-{
-  "ip_address" : "malformed ip address"
-}
-```
-
-This indexed IP address does not throw an error because `ignore_malformed` is set to true. 
-
-You can query the index using the following request:
-
-```json
-GET /test-index/_search
-```
+| Setting | Default value | Allowed value | Type | Description |
+|-|-|-|-|-|
+| `index.mapping.nested_fields.limit` | 50 | [0,) | Dynamic | Limits the maximum number of nested fields that can be defined in an index mapping. |
```

Comment:
Does `[0,)` mean that any value above `0` is allowed?

Reply:
@mgodwan Please see question above.